pkg/test/istio.io/tasks/security/secure-gateway-sds/mtls-go-example.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
Same here, and in your other scripts.
Please replace all your hand-written scripts with copy-and-paste from the doc's shell snippets.

Reply:
Sorry I put wrong link in the PR description. The correct link is https://preliminary.istio.io/docs/tasks/traffic-management/ingress/secure-ingress-sds/.
These scripts are copy-and-paste from doc's shell snippets.